docker-compose.yml

```diff
@@ -72,6 +72,17 @@ services:
       -rpcallowip=::/0
       -deprecatedrpc=signrawtransaction
 
+  k6:
+    build: ./tests/stress
+    depends_on:
+      - poet-node
+    links:
```

Comment:
links are no longer required. If you remove the links array, you should be able to docker-compose exec into the k6 container and ping poet-node.

Reply:
awesome, I will test